### ISSUE TITLE
Frequent formula updates

### ISSUE THREAD

**Issue by JordonPhillips** [Homebrew/homebrew-core#3369]
The AWS CLI has had some customers asking for us to maintain the homebrew formula so that it can be always up to date. Some of that discussion is encapsulated in aws/aws-cli#727.

The biggest question for us is update cadence. We update several times a week, so we would need to make sure that we update the formula at the same time we cut a release. Some amount of delay is acceptable of course, but we'd like to get it out as fast as possible.

Is there any sort of automation that can be taken advantage of? Based on the [formula history](https://github.com/Homebrew/homebrew-core/commits/master/Formula/awscli.rb) it seems that there is either something to that effect in play already or otherwise @ilovezfs has taken charge of getting things out.

So to summarize, what can we do to make sure that our customers can update via homebrew ASAP and what kind of lag can we expect?

**Comment by MikeMcQuaid** [comment]
> Is there any sort of automation that can be taken advantage of?

@JordonPhillips You could set `HOMEBREW_DEVELOPER=1` in your environment and run `brew bump-formula-pr` (https://github.com/Homebrew/brew/blob/master/Library/Homebrew/dev-cmd/bump-formula-pr.rb) to create a pull request as part of your release process.

> So to summarize, what can we do to make sure that our customers can update via homebrew ASAP and what kind of lag can we expect?

It's likely if the only change is the URL and SHA256 then it'd be pulled within a 24h (perhaps even within 15m depending on activity). However, we are volunteers doing this in our spare time so we cannot provide any guarantees.

**Comment by JordonPhillips** [comment]
@MikeMcQuaid Thanks! I totally get that there can be no guarantees. As a homebrew user myself, I greatly appreciate the work y'all do.

If this is a manual process rather than automated, should we pursue updating here? We could potentially be pushing releases every day, I don't want to consume all your bandwidth.

**Comment by MikeMcQuaid** [comment]
Thanks for the kind words ❤️ 

> If this is a manual process rather than automated, should we pursue updating here? We could potentially be pushing releases every day, I don't want to consume all your bandwidth.

It's fine to update here potentially daily; pulling changes like this is pretty low maintainance.

**Comment by JordonPhillips** [comment]
Awesome, I think that's all the info I need!
